src/content/docs/mobile-monitoring/mobile-monitoring-ui/network-pages/http-errors-network-failure-analysis.mdx

```diff
@@ -229,7 +229,6 @@ To view details about an error trace on the **Errors** page, select its request
 
 * View the response body.
 * Share the error details with others by email.
```

Comment:
@austin-schaefer, so it looks like you're not sending users to the errors inbox Jira integration. Is that because it doesn't relate to this or that we want to keep this simple until we get more integrations?

Reply:
Honestly, it's because I didn't have the product knowledge to address this with confidence.